apps/happy-blocks/block-library/support-content-footer/style.scss

```diff
@@ -232,7 +232,7 @@ $system-font-family: -apple-system, BlinkMacSystemFont, "Noto Sans", "Segoe UI",
 
 		.support-content-links {
 			a {
-				display: inline-block;
+				display: block;
```

Comment:
I believe we can remove this, it was solved https://github.com/Automattic/wp-calypso/pull/80758 could you please double check?

Reply:
Ok, removed this!